gpflow/optimizers/natgrad.py

```diff
@@ -39,114 +47,212 @@ class XiTransform(metaclass=abc.ABCMeta):
     the parameters pairs are always of shape (N, D) and (D, N, N).
     """
 
+    @staticmethod
     @abc.abstractmethod
-    def meanvarsqrt_to_xi(self, mean, varsqrt):
+    def meanvarsqrt_to_xi(mean, varsqrt):
         """
         Transforms the parameter `mean` and `varsqrt` to `xi1`, `xi2`
 
         :param mean: the mean parameter (N, D)
         :param varsqrt: the varsqrt parameter (D, N, N)
         :return: tuple (xi1, xi2), the xi parameters (N, D), (D, N, N)
         """
-        pass  # pragma: no cover
 
+    @staticmethod
     @abc.abstractmethod
-    def xi_to_meanvarsqrt(self, xi1, xi2):
+    def xi_to_meanvarsqrt(xi1, xi2):
         """
         Transforms the parameter `xi1`, `xi2` to `mean`, `varsqrt`
 
         :param xi1: the ξ₁ parameter
         :param xi2: the ξ₂ parameter
         :return: tuple (mean, varsqrt), the meanvarsqrt parameters
         """
-        pass  # pragma: no cover
 
+    @staticmethod
     @abc.abstractmethod
-    def naturals_to_xi(self, nat1, nat2):
+    def naturals_to_xi(nat1, nat2):
         """
         Applies the transform so that `nat1`, `nat2` is mapped to `xi1`, `xi2`
 
         :param nat1: the θ₁ parameter
         :param nat2: the θ₂ parameter
         :return: tuple `xi1`, `xi2`
         """
-        pass  # pragma: no cover
+
+
+class XiNat(XiTransform):
```

Comment:
I'm not sure. What happened here? :)

Reply:
I moved it to the top of the file so it could be used within the NaturalGradient class, and I changed the methods to be staticmethods as discussed with @condnsdmatters.